model/src/main/kotlin/Package.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.model
 
 import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
```

Comment:
Commit message:
* Maybe "arbitrary key value pairs" instead of "arbitrary data"
* Typo in "particular"